src/flow_information.rs

```diff
@@ -6,6 +6,8 @@ use crate::packet::Packet;
 /// It contains forward and backward packet's.
 #[derive(Debug, Default, Deserialize, Serialize)]
 pub struct FlowInformation {
+    /// SNI field
+    pub sni: String,
```

Comment:
I would prefer Option<&str>:

Optional to be able to easily handle the missing values
&str to reduce the payload (String gathers a lot of methods...useless here).

Reply:
you 're right

Reply:
we keep String. 
the type becomes Option<String>